staff/edgar-frias-jimenez/spotify-template/desktop.scss

```diff
@@ -0,0 +1,277 @@
+              transition-duration: .2s;
+              transition-timing-function: linear;
+              color: #b3b3b3;
+
+              &:focus,
+              &:hover {
+                color: white;
+              }
+
+              span {
+                margin-left: 12px;
+              }
+            }
+
+            &--selected {
+              position: relative;
+
+              &::after {
+                content: '';
+                display: block;
+                position: absolute;
+                left: 0;
+                top: 5px;
+                bottom: 5px;
+                width: 4px;
+                background-color: $green;
+              }
+
+              a {
+                color: white;
+              }
+            }
+          }
+        }
+      }
+    }
+
+    .content {
+      display: flex;
+      flex-direction: row;
+      justify-content: flex-start;
+      flex-grow: 1;
+
+      &__container {
+        padding: 0 32px;
+        margin-left: 230px;
+        max-width: 1480px;
+        width: calc(100% - 230px);
+      }
+    }
+
+    header {
+      ul {
+        display: block;
+        padding: 20px 0;
+        text-align: center;
+        width: 100%;
+
+        li {
+          display: inline-block;
+          font-size: 12px;
+          line-height: 16px;
+          letter-spacing: .015em;
+          color: #b3b3b3;
+          text-transform: uppercase;
+          letter-spacing: .16em;
+          color: #fff;
+          opacity: .6;
+          position: relative;
+          margin: 10px;
+          padding: 5px 10px;
+          transition: opacity .2s linear;
+
+          &:hover {
+            cursor: pointer;
+            opacity: 1;
+          }
+
+          &.selected {
+            &:before {
+              content: "";
+              height: 2px;
+              background-color: $green;
+              width: 30px;
+              position: absolute;
+              display: block;
+              left: 50%;
+              bottom: 0;
+              margin-left: -15px
+            }
+          }
+        }
+      }
+    }
+
+    .main-content {
+      .playlist-container {
+        .playlist__title {
+          display: flex;
+          align-items: center;
+          justify-content: space-between;
+          font-size: 36px;
+          line-height: 44px;
+          margin: 24px 0;
+
+          a {
+            display: block;
+            font-size: 12px;
+            line-height: 16px;
+            text-transform: uppercase;
+            opacity: .6;
+            text-decoration: none;
+          }
+        }
+
+        .playlist {
+          display: flex;
+          flex-flow: row wrap;
+          justify-content: flex-start;
+          align-items: center;
+
+          &__item {
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            padding: 0 8px;
+            width: auto;
+
+            &__image {
```

Comment:
@edfrias what a nice job you just did! Remember that BEM only accepter two underscores, so image should be `playlist__item-image`. Please update it and informs us once it's done! Keep it up!

Reply:
Ops! You're right mate! I'll update on a next pr! Thanks!